x-pack/solutions/observability/plugins/synthetics/public/apps/synthetics/components/monitors_page/overview/overview/metric_item.tsx

```diff
@@ -4,55 +4,48 @@
  * 2.0; you may not use this file except in compliance with the Elastic License
  * 2.0.
  */
+import { Chart, DARK_THEME, Metric, MetricTrendShape, Settings } from '@elastic/charts';
+import { EuiPanel, EuiSpacer, EuiThemeComputed, useEuiTheme } from '@elastic/eui';
+import { css } from '@emotion/react';
 import { i18n } from '@kbn/i18n';
-import React, { useState } from 'react';
 import { FormattedMessage } from '@kbn/i18n-react';
-import { css } from '@emotion/react';
-import { Chart, Settings, Metric, MetricTrendShape } from '@elastic/charts';
-import { EuiPanel, EuiSpacer } from '@elastic/eui';
-import { DARK_THEME } from '@elastic/charts';
-import { useTheme } from '@kbn/observability-shared-plugin/public';
 import moment from 'moment';
-import { useSelector, useDispatch } from 'react-redux';
-
-import { FlyoutParamProps } from './types';
-import { MetricItemBody } from './metric_item/metric_item_body';
+import React, { useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { OverviewStatusMetaData } from '../../../../../../../common/runtime_types';
+import { useLocationName, useStatusByLocationOverview } from '../../../../hooks';
 import {
   selectErrorPopoverState,
   selectOverviewTrends,
   toggleErrorPopoverOpen,
 } from '../../../../state';
-import { useLocationName, useStatusByLocationOverview } from '../../../../hooks';
-import { formatDuration } from '../../../../utils/formatting';
-import { OverviewStatusMetaData } from '../../../../../../../common/runtime_types';
-import { ActionsPopover } from './actions_popover';
 import {
   hideTestNowFlyoutAction,
   manualTestRunInProgressSelector,
   toggleTestNowFlyoutAction,
 } from '../../../../state/manual_test_runs';
-import { MetricItemIcon } from './metric_item_icon';
+import { formatDuration } from '../../../../utils/formatting';
+import { ActionsPopover } from './actions_popover';
+import { MetricItemBody } from './metric_item/metric_item_body';
 import { MetricItemExtra } from './metric_item/metric_item_extra';
+import { MetricItemIcon } from './metric_item_icon';
+import { FlyoutParamProps } from './types';
 
 const METRIC_ITEM_HEIGHT = 160;
 
-export const getColor = (
-  theme: ReturnType<typeof useTheme>,
-  isEnabled: boolean,
-  status?: string
-) => {
+export const getColor = (euiTheme: EuiThemeComputed, isEnabled: boolean, status?: string) => {
   if (!isEnabled) {
-    return theme.eui.euiColorLightestShade;
+    return euiTheme.colors.lightestShade;
   }
   switch (status) {
     case 'down':
-      return theme.eui.euiColorVis9_behindText;
+      return euiTheme.colors.vis.euiColorVis6;
```

Comment:
i wonder if we should just use danger here

Reply:
probably, I don't mind

Reply:
These updates change the color output for Amsterdam. If this is unwanted, we'll need to conditionally update and keep previous values for Amsterdam.